robocorp-code/codegen/commands.py

```diff
@@ -565,6 +565,20 @@ def __init__(
         hide_from_command_palette=True,
         icon="$(type-hierarchy-sub)",
     ),
+    Command(
+        "robocorp.convertProject",
+        "Convert file to Robocorp Project",
```

Comment:
As mentioned, we probably could have a different name to make the purpose clearer (let's do this over slack).